README.md

```diff
@@ -24,7 +24,7 @@ npm install tinycolor2
 ## Use
 
 ```ts
```

Comment:
Nit: we could update these to "js" instead of "ts". Not sure if that matters in practice for these cases.